blockly-core/core/ui/contract_editor/contract_editor.js

```diff
@@ -67,15 +92,15 @@ Blockly.ContractEditor.EXAMPLE_BLOCK_TYPE = 'functional_example';
 Blockly.ContractEditor.EXAMPLE_BLOCK_ACTUAL_INPUT_NAME = 'ACTUAL';
 
 Blockly.ContractEditor.typesToColorsHSV = {
-  'none': [0, 0, 0.6],
+  // 'none': [0, 0, 0.6], // not available to users
   'Number': [192, 1.00, 0.99], // 00ccff
   'string': [180, 1.00, 0.60], // 0099999
   'image': [285, 1.00, 0.80], // 9900cc
   'boolean': [90, 1.00, 0.4] // 336600
 };
```

Comment:
Re capitalization, not sure, I cargo-cult-copied it from the functional block internals code, I think with some vague concern that existing blocks would need to be cleaned up / there might be some required JavaScript type match-up there. Other thinking was that eventually these will be localized and internal keywords only.

Worth figuring out and normalizing now, though, especially if we'll start storing types in saved user XML it'd be harder to clean up later. I'll single-point-of-truth this all and see if I can do the capitalization replace